h_program-lang/Parse_info.mli

```diff
@@ -131,13 +131,19 @@ val lexical_error: string -> Lexing.lexbuf -> unit
 val fake_token_location : token_location
 (*e: signature [[Parse_info.fake_token_location]] *)
 (*s: signature [[Parse_info.fake_info]] *)
-val fake_info : string -> t
+val unsafe_fake_info : string -> t
```

Comment:
You probably need to add more comments in the .mli, maybe something like
"The safe (no unsafe_ prefix) variants below are called "safe" because they
require an existing token to attach to, and so token_location_of_info will work also
on those fake tokens."